autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/sync/CredentialsSync.kt

```diff
@@ -187,4 +214,22 @@ class CredentialsSync @Inject constructor(
             lastUpdatedMillis = details.lastUpdatedMillis,
         )
     }
+
+    private fun List<LoginCredentialEntry>.filterInvalidItems(): List<LoginCredentialEntry> {
```

Comment:
`filterInvalidItems` it's hard to understand from the function name whether it's going to return only valid or only invalid entries.

`list.filter()` returns the ones which match the predicate, so it kinda sounds like this function should return only invalid ones from the function name, but i can see it's the opposite.

can we rename this to `validItems()` or `removeInvalidItems()` or such?